docs/getting_started/resources/index.md

```diff
@@ -0,0 +1,3 @@
+# BIDS Resources
+
+The following collection of resources have been compiled to help you get started with BIDS. The links section has links to software tools to help you create and work with BIDS datasets, glossary and dependencies to help you get accustomed to common "jargon" and tools associated with working with BIDS, and some publications and talks covering recent work using BIDS organization and tools. 
```

Comment:
```suggestion
The following collection of resources have been compiled to help you get started with BIDS. The links section has links to software tools to help you create and work with BIDS datasets, glossary and dependencies to help you get accustomed to common "jargon" and tools associated with working with BIDS, and some publications and talks covering recent work using BIDS organization and tools. 
```
to simplify and clarify this super long sentence. 
- [ ] separate the glossary and dependencies clauses (they're unrelated) - maybe mention dependencies after Links to software tools
- [ ] read out loud and see where you can remove a few words e.g. "to help you create" could be "for creating" -- "to help you" is repeated.

Reply:
updated